rafthttp/pipeline.go

```diff
@@ -16,13 +16,13 @@ package rafthttp
 
 import (
 	"bytes"
+	"context"
```

Comment:
should we import `"golang.org/x/net/context"` instead of `context`? @xiang90 @heyitsanthony .